vendor/bat-native-ads/src/bat/ads/internal/ads_serve.cc

```diff
@@ -10,9 +10,10 @@
 #include "bat/ads/internal/bundle.h"
 #include "bat/ads/internal/logging.h"
 
-#include "base/rand_util.h"
 #include "base/time/time.h"
 
+#include "vendor/brave_base/random.h"
```

Comment:
I thought we were going with `brave_base/random.h` for include path? https://bravesoftware.slack.com/archives/C7VLGSR55/p1557934054241500?thread_ts=1557867482.222100&cid=C7VLGSR55

so include_dirs would be "." here https://github.com/brave/brave-core/pull/2446/files#diff-db25724f33e3b882a36926fbfcfe8926R6

Reply:
Forgot about that, done.